build-parent/pom.xml

```diff
@@ -74,6 +74,10 @@
         <version.quarkus.cucumber>0.6.0</version.quarkus.cucumber>
         <version.quarkus.cucumber.java>7.4.1</version.quarkus.cucumber.java>
 
+        <!-- io.fabric8:docker-maven-plugin -->
+        <version.fabric8.docker.maven.plugin>0.44.0</version.fabric8.docker.maven.plugin>
+        <docker.platforms>linux/amd64,linux/arm64</docker.platforms>
```

Comment:
I: Fabric8 will only build your local machines architecture when building on `mvn clean install` so safe to list both platforms here.

Reply:
Q: will this build both platforms for the downstream project?